### ISSUE TITLE
Installation fails on 14.04 at: add-apt-repository: error: no such option: -u

### ISSUE THREAD

**Issue by krisjace** [anbox/anbox#30]
Make sure you are running the latest version of Anbox before reporting an issue.

**Anbox release (`anbox version`) and system info (`anbox system-info`):**

**Description of the problem:**

installation scripts stops with the error message:
add-apt-repository: error: no such option: -u

**Expected:**

**Additional info:**

**Comment by xdtianyu** [comment]
Hi, I'm facing the same problem. You can remove the `-u` from [installer.sh#L122](https://github.com/anbox/anbox-installer/blob/master/installer.sh#L122).

1\. Download script file.

`wget https://raw.githubusercontent.com/anbox/anbox-installer/master/installer.sh -O anbox-installer`

2\. Replace

 `sudo add-apt-repository -y -u 'ppa:morphis/anbox-support'` 

with 

`sudo add-apt-repository -y 'ppa:morphis/anbox-support' `

3\. Run `anbox-installer` again

```
chmod +x anbox-installer
./anbox-installer
```

**Comment by krisjace** [comment]
Yes I actually did this, but I still want to report this so they can update their script for the rest of us.

**Comment by krisjace** [comment]
Just to clarify, I was able to fix that one as well, but they should update the script.
*the script didn't apt-get update after adding the repo, thus could not find anbox-modules-dkms.
After manually apt-get update, and re-running the script it was found.

**Comment by krisjace** [comment]
further down the road:

> + sudo snap install --edge --devmode anbox
> ZOE ERROR (from /usr/lib/snap/snap): zoeParseOptions: unknown option (--edge)
> ZOE library version 2006-07-28
> 
after removing "--edge" I got:

> + sudo snap install --devmode anbox
> ZOE ERROR (from /usr/lib/snap/snap): zoeParseOptions: unknown option (--devmode)
> ZOE library version 2006-07-28
> 

and after also removing "--devmode" I ended up with:

> + sudo snap install anbox
> ZOE ERROR (from /usr/lib/snap/snap): error opening parameter file
> ZOE library version 2006-07-28
>

**Comment by krisjace** [comment]
Again, this could been resolved by manually running:

> $ sudo apt-get install snapd

But I think the installation script should do that.

**Comment by krisjace** [comment]
Next stop on the instalation:

> + '[' '!' -e /etc/modules-load.d/anbox.conf ']'
> + sudo modprobe binder_linux
> + sudo modprobe ashmem_linux
> + snap list anbox
> + sudo snap refresh anbox
> error: cannot refresh "anbox": cannot find snap "anbox"
> 

is the anbox snap not published for Ubuntu 14.04?

**Comment by morphis** [comment]
@krisjace The snap is published indepdently of any Ubuntu or distribution version. If you have snaps then you have all of them. The only problem here could be that you're either on i386 (which we don't support yet) or you hit another bug in the script. Can you check if you have the snap already installed with `snap list` and what `snap info anbox` says?

**Comment by krisjace** [comment]
I'm on x64, output of `snap list`:

> No snaps are installed yet. Try "snap install hello-world".

output of `snap info anbox`:
```
name:      anbox
summary:   "Android in a Box"
publisher: morphis
description: |
  Runtime for Android applications which runs a full Android system
  in a container using Linux namespaces (user, ipc, net, mount) to
  separate the Android system fully from the host.
  
channels:            
  edge:   1-dev (15) 357MB devmode

```

**Comment by krisjace** [comment]
and just as a double checking practice, `sudo apt-get install snapd:amd64`:

> Reading package lists... Done
> Building dependency tree       
> Reading state information... Done
> snapd is already the newest version.
> 0 upgraded, 0 newly installed, 0 to remove and 8 not upgraded.
> 

so yes, surely it's not i386 case

**Comment by morphis** [comment]
Could be that this is fixed with https://github.com/anbox/anbox-installer/pull/1

**Comment by krisjace** [comment]
I think I am using the fixed version, because inside my script:
```
if snap list anbox >/dev/null 2>&1; then
	 sudo snap refresh anbox
else
	 sudo snap install --edge --devmode anbox
fi

```
and I just wgotten the script today, so I guess it is up to date

**Comment by jzyken** [comment]
2. Replace

sudo add-apt-repository -y -u 'ppa:morphis/anbox-support'

with

sudo add-apt-repository -y 'ppa:morphis/anbox-support'
sudo apt update
